### DISCUSSION TITLE
Any plan to support React?

### DISCUSSION THREAD

**Discussion by kawpii** [saadeghi/daisyui#451] [Q&A] [answered]
I know there is examples in the repo using React but it seems they use non interactive elements like buttons

Any plans to support interactive elements like Dropdowns, drawers, modals with React support?

**Comment by saadeghi** [comment, marked as answer]
daisyUI is just CSS.  
So everything works in React, Vue, Svelte, etc...

For React, you just need to replace a few things in your HTML: Replace `class` with `className` and make sure all self closing tags have slash at the end (like `<input />` instead of `<input>`)

**Reply by kawpii** [reply]
Since daisyUI is CSS only one have to write JS code in a React compatible way for interactive components. ex: modal

**Reply by kawpii** [reply]
I actually tried out modal on you website and checked keyboard navigation when modal is open, unluckily I can navigate to other page element when modal is open.

**Reply by kawpii** [reply]
Any way I am happy someone is trying to make CSS only components than bloated components with CSS in JS.
